tools/targets/lint.py

```diff
@@ -82,7 +82,7 @@ def check_inherits(dict):
                       "LOWPOWERTIMER", "PORTIN", "PORTINOUT", "PORTOUT",
                       "PWMOUT", "RTC", "TRNG","SERIAL", "SERIAL_ASYNCH",
                       "SERIAL_FC", "SLEEP", "SPI", "SPI_ASYNCH", "SPISLAVE",
-                      "STORAGE"]
+                      "STORAGE", "STCLK_OFF_DURING_SLEEP"]
```

Comment:
looking at this again, is this a proper placement for this? Is it `device_` ? Wouldnt this be rather a targets config? `MBED_CONF_TARGET_STCLK.....` - better place/definition for this?

Reply:
I think it's a device characteristic not a config.